AI/VCAI/VCAI.h

```diff
@@ -138,7 +141,10 @@ class VCAI : public CAdventureAI
 
 	friend class FuzzyHelper;
 
+	std::map<TeleportChannelID, shared_ptr<TeleportChannel> > knownTeleportChannels;
 	std::map<const CGObjectInstance *, const CGObjectInstance *> knownSubterraneanGates;
+	ObjectInstanceID destinationTeleport;

```

Comment:
> Currently it is impossible

Sadly it's kind a possible because there is way to unlock UI and use save via hotkey.
Though as I get it's shouldn't be that way.